Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml

```diff
@@ -0,0 +1,33 @@
+commonfields:
+  id: CompareIndicators
+  version: -1
+name: CompareIndicators
+fromversion: 6.0.0
+script: ''
+type: python
+tags: []
+comment: Find the differences between two indicators lists.
```

Comment:
just add a statement for what indicators are supported currently

Reply:
It supports all type of indicators